corehq/apps/hqcase/tests/test_case_api_permissions.py

```diff
@@ -0,0 +1,213 @@
+import uuid
+from contextlib import contextmanager
+
+from django.test import TestCase
+from django.urls import reverse
+
+from casexml.apps.case.mock import CaseBlock
+
+from corehq import privileges
+from corehq.apps.commtrack.tests.util import bootstrap_domain
+from corehq.apps.es.tests.utils import (
+    case_search_es_setup,
+    case_search_es_teardown,
+    es_test,
+)
```

Comment:
You'll also need the `user_adapter` (for telling the `es_test` decorator you want it bootstrapped for you) and the `manager` (in order to force-refresh the index, see comments below).

```suggestion
from corehq.apps.es.client import manager
from corehq.apps.es.tests.utils import (
    case_search_es_setup,
    case_search_es_teardown,
    es_test,
)
from corehq.apps.es.users import user_adapter
```